### ISSUE TITLE
KeyError: 'items' when trying to like timeline

### ISSUE THREAD

**Issue by TiasTias** [ohld/igbot#686]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like.

---

### Before submitting an issue, make sure you have:
- [x] Updated to the latest version
- [x] Read the [README](https://github.com/instagrambot/instabot/blob/master/README.md) and [docs](https://github.com/instagrambot/instabot/tree/master/docs)
- [x] [Searched](https://github.com/instagrambot/instabot/search?type=Issues) the bugtracker for similar issues including **closed** ones

### Purpose of your issue?
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections requests more details for particular types of issues, you can remove any section (the contents between the triple ---) not applicable to your issue.

---

### For a *bug report*, you **must** include the Python version used, *code* that will reproduce the error, and the *error log/traceback*.

Paste the output of ``python -V`` here: Python 2.7.10

Code:

```python
def get_timeline_medias(self, filtration=True):
    if not self.api.get_timeline_feed():
        self.logger.warning("Error while getting timeline feed.")
        return []
    return self.filter_medias(self.api.last_json["items"], filtration)
```

Error/Debug Log:

```python
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "ultimate.py", line 44, in like_timeline
    bot.like_timeline(amount=300 // 24)
  File "build/bdist.macosx-10.13-intel/egg/instabot/bot/bot.py", line 422, in like_timeline
    return like_timeline(self, amount)
  File "build/bdist.macosx-10.13-intel/egg/instabot/bot/bot_like.py", line 61, in like_timeline
    medias = self.get_timeline_medias()[:amount]
  File "build/bdist.macosx-10.13-intel/egg/instabot/bot/bot.py", line 309, in get_timeline_medias
    return get_timeline_medias(self)
  File "build/bdist.macosx-10.13-intel/egg/instabot/bot/bot_get.py", line 46, in get_timeline_medias
    return self.filter_medias(self.api.last_json["items"], filtration)
KeyError: 'items'
```

---

### Describe your issue

Since the beginning of today I have been getting this error. I did not add any code. I just simply ... $python setup.py build... Then $Python setup.py install ... In the code root dir because my instabot was not taking into account the updated values for max_followers_to_follow=10000 and filter_business_accounts=False when I was running the ultimate schedule script. I recloned the repo and installed via pip but it still producers this error. 

Tried another fresh install and even running the multi_script_CLI.py it crashes because of KeyError: 'items' when trying to like timeline. It appears to like from hashtag but is super buggy. See screenshot.

<img width="1022" alt="screen shot 2018-10-18 at 11 33 03" src="https://user-images.githubusercontent.com/43808248/47145603-62f12e00-d2ca-11e8-9061-5a9789f1321d.png">

**Comment by aferook** [comment]
I had the same error, #689 solves my problem

**Comment by TiasTias** [comment]
> I had the same error, #689 solves my problem

Okay, so I changed that one line 46 to 
```python
   return self.filter_medias(self.api.last_json["feed_items"], filtration)
```
and restarted the bot but it still doesn't work. Is there an extra step I am missing?

**Comment by aferook** [comment]
Is there any error message?

**Comment by TiasTias** [comment]
Yes, the KeyError: 'items' .

 I think there may be something wrong with my environment though since I am getting these other strange behaviours such as this unnecessary filtering of medias for instance (see the image in the original post). I don't know how to reset it other than deleting the files and re-downloaded the repo but that didn't change anything.

**Comment by sudoguy** [comment]
Fixed #689 . Thanks to @aferook
